1.cellprofiler_ic/README.md

```diff
@@ -0,0 +1,15 @@
+## Perform illumination correction and save corrected images
+
+In this module, we perform illumination correction (IC) on the images for each plate and save the corrected images into new folders. 
+The images are saved as 16-bit depth, which is the same as the raw data.
+
+## Run the `nf1_ic` notebook
+
+To calculate and apply an IC function on each channel, run the [nf1_ic.ipynb](dnf1_ic.ipynb) notebook as a python script using the code block below:
+
+```bash
+# Run this script in terminal
+conda activate nf1_cellpainting_data
+cd 0.cellprofiler_ic
```

Comment:
Could these first two lines be included within `nf1_ic.sh`, simplifying this step in the readme to solely running the script?

Reply:
I just moved these commands into the `sh` file! I agree with the simplification!